app/controllers/application_controller.rb

```diff
@@ -46,6 +47,11 @@ def create_user_event(event_type, user = current_user)
 
   private
 
+  def disable_caching
+    response.headers['Cache-Control'] = 'no-store'
+    response.headers['Pragma'] = 'no-cache'
```

Comment:
OWASP recommends some additional directives as well: https://www.owasp.org/index.php/OWASP_Application_Security_FAQ#Am_I_totally_safe_with_these_directives.3F

`pre-check=0, post-check=0, max-age=0, and s-maxage=0`